libs/community/tests/integration_tests/vectorstores/test_cratedb.py

```diff
@@ -232,7 +232,7 @@ def test_cratedb_with_metadatas_with_scores() -> None:
         pre_delete_collection=True,
     )
     output = docsearch.similarity_search_with_score("foo", k=1)
-    assert output == [(Document(page_content="foo", metadata={"page": "0"}), 2.0)]
+    assert output == [(Document(page_content="foo", metadata={"page": "0"}), 1.0)]
```

Comment:
This spot, and others, perfectly demonstrates that `vector_similarity()` produces sensible values, now ranging between 0.0 and 1.0.

Reply:
Yes, `_score` is unbounded, which will not work in this case